spike/CodeReaderClass.py

```diff
@@ -0,0 +1,151 @@
+class CodeParser():
+
+    #Parsing variables
+    PORT = ''
+    OPERATION = ''
+    NUMBER = ''
+
+    #Parsing libraries (or "keywords" if you wanna not sound like an arrogant prick)
+    OPERATIONS = ['+', '-', '*', '/', '=']
+    NUMBERS = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9']
+    WHITESPACE = ' '
+    PORTS = ["THROTTLE", "TURN"]
+
+    #Initialize the CodeParser
+    def __init__(self):
+        self.PATH = "Racer.txt"
```

Comment:
Have `__init__` take a variable `path` instead of hard coding it. Then we can more easily change the file it parses which will allow for better unit tests.